test/fixtures/past_events.yml

```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+# This model initially had no columns defined. If you add columns to the
+# model remove the '{}' from the fixture names and add the columns immediately
+# below each fixture, per the syntax in the comments below
+#
+one: {}
```

Comment:
need to specify the values as it is referenced in the test/controllers/past_events_controller_test.rb

Reply:
I will insert test cases with Rspec soon